shared/AppInsightsCore/Tests/Unit/src/ApplicationInsightsCore.Tests.ts

```diff
@@ -338,6 +338,49 @@ export class ApplicationInsightsCoreTests extends AITestClass {
             }
         });
 
+        this.testCase({
+            name: "Initialization: channels adds and initialize with offline channel with channel config",
+            useFakeTimers: true,
+            test: () => {
+                let offlineChannelPlugin = new TestOfflineChannelPlugin();
+
+                let channelPlugin = new TestChannelPlugin();
+              
+
+                const appInsightsCore = new AppInsightsCore();
+                appInsightsCore.initialize(
+                    { instrumentationKey: "09465199-12AA-4124-817F-544738CC7C41", channels: [[offlineChannelPlugin, channelPlugin]] },
```

Comment:
Do we really need a real looking key? using a dummy key for testing is probably better to avoid any possible clash or exposure of internal testing keys

Reply:
changed the ikey to "testIkey"